src/components/savedetailmodal.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {defineMessages, injectIntl, intlShape} from 'react-intl';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import FloatingActionButton from 'material-ui/FloatingActionButton';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
```

Comment:
Can we use the SDK Button class instead? In case we need tooltips later on? Instead of FloatingActionButton and FlatButton